tests/baselines/reference/arrayFilter.errors.txt

```diff
@@ -0,0 +1,15 @@
+tests/cases/compiler/arrayFilter.ts(7,12): error TS2345: Argument of type '(x: { name: string; }) => string' is not assignable to parameter of type '(value: { name: string; }, index: number, array: { name: string; }[]) => boolean'.
+  Type 'string' is not assignable to type 'boolean'.
+
+
+==== tests/cases/compiler/arrayFilter.ts (1 errors) ====
+    var foo = [
+       { name: 'bar' },
+       { name: null },
+       { name: 'baz' }
+    ]
+    
+    foo.filter(x => x.name); //should accepted all possible types not only boolean!
+               ~~~~~~~~~~~
+!!! error TS2345: Argument of type '(x: { name: string; }) => string' is not assignable to parameter of type '(value: { name: string; }, index: number, array: { name: string; }[]) => boolean'.

```

Comment:
@RyanCavanaugh btw, I made another branch and did exactly the flow that you said ^^, I run jake runtests for second time and it was passed, but still there was that file(arrayFilter.errors.txt) in my git which I thought should be part of my commit! anyway, I commit without that! but I want to know is it correct?! I mean shouldn't be deleted by jake runtests!?